modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -1,15 +1,84 @@
 
-You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
+Connection issues between your on-premises Mule runtime engine and the Runtime Manager agent might cause discrepancies, such as metrics not appearing in the server monitoring dashboard in Runtime Manager.
+
+== Troubleshoot Connection Issues Between the Agent and Mule
+
+To troubleshoot connection issues:
+
+. Check the JVM you are using.
++
+If you are using the IBM JVM, you must install the truststore manually because this truststore requires different parameters than Oracle JVM or OpenJDK.
++
+See xref:servers-create.adoc#ibm-jvm[IBM JVM].
+. Ensure you are using the latest agent version.
++
+.. Check the agent version.
++
+In the Runtime Manager navigation menu, select *Servers* and then click the server name.
+The agent version appears on the *Dashboard* page:
++
+.The arrow shows the Runtime Manager agent version on the server *Dashboard* page.
+image::rtm-agent-version.png[Runtime Manager agent version on the server Dashboard page]
+.. Check the xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes] to determine the latest agent version compatible with your Mule version.
+.. Update the agent, if necessary.
++
+See xref:installing-and-configuring-runtime-manager-agent.adoc#update-the-agent[Update the Agent].
+. Verify that the license and certificate are not expired.
++
+.. In the Runtime Manager navigation menu, select *Servers* and then click the server name.
+The expiration dates appear on the *Dashboard* page:
++
+.The arrow shows the expiration dates on the server *Dashboard* page.
+image::rtm-server-expirations.png[Expiration dates on the server Dashboard page]
++
+//DASHBOARD SHARED 
+include::partial$cluster-server.adoc[tag=viewServerinClusterOrGroup]
++
+* If the license has expired, see xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License].
+* If the certificate has expired, see xref:servers-cert-renewal.adoc[Renew Server Certificates].
+. Verify connectivity with all required hosts.
+.. Ensure that your allowlist includes the required hostnames.
++
+See xref:rtm-agent-whitelists.adoc[Ports, IP Addresses, and Hostnames to Whitelist].
+.. Verify that your firewall or proxy server is not intercepting the Runtime Manager certificate.
++ 
+See xref:rtm-agent-proxy-config,adoc#verify-that-the-proxy-server-does-not-modify-the-runtime-manager-certificate[Verify That the Proxy Server Does Not Modify the Runtime Manager Certificate].
+. Verify that the `mule-agent.yml` file is not corrupted or empty.
++
+If the `<MULE_HOME>/conf/mule-agent.yml` file is empty or contains unknown characters:
```

Comment:
Asking with a newbie user hat on: is it explained somewhere how to determine the "MULE_HOME" path? Is the path stored as an environment variable?

How does one know if the agent file is corrupted? Is this a known issue?

I notice that later (line 80) you say $MULE_HOME as opposed to <MULE_HOME>. Not a big deal, but consider changing for consistency.

Reply:
Added some clarification - they will have encountered this when they installed Mule.... over there in the Mule runtime repo.